genai-perf/docs/goodput_tutorial.md

```diff
@@ -0,0 +1,134 @@
+<!--
+Copyright (c) 2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions
+are met:
+ * Redistributions of source code must retain the above copyright
+   notice, this list of conditions and the following disclaimer.
+ * Redistributions in binary form must reproduce the above copyright
+   notice, this list of conditions and the following disclaimer in the
+   documentation and/or other materials provided with the distribution.
+ * Neither the name of NVIDIA CORPORATION nor the names of its
+   contributors may be used to endorse or promote products derived
+   from this software without specific prior written permission.
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+
+# Context
+
+Goodput, defined as the number of completed requests per second that meet the Service Level Objectives (SLOs), provides an enhanced measure of AI serving performance by accounting for both cost efficiency and user satisfaction.
+
+GenAI-Perf is continually evolving to meet the demands of next-generation GenAI workflows. We are excited to introduce our latest feature: goodput support for measuring Service Level Objectives.
+
+This new feature empowers you to more precisely evaluate the efficiency and effectiveness of your AI models and services. By measuring goodput, GenAI-Perf delivers deeper insights into how well your services adhere to your defined SLOs, enabling you to optimize both performance and cost efficiency while prioritizing the user experience.
+
+# Tutorials
+
+This is a tutorial on how to use our goodput feature by examples.
+
+## Examples
+
+- [LLM Examples](#LLM)
+
+- [Embedding Model Examples](#embeddings)
+
+## Profile GPT2 Goodput<a id="LLM"></a>
```

Comment:
You should aim to be consistent here. Since you use LLM throughout, the title should be Profile LLM Goodput so that it's obvious that this is the "LLM Examples" section.

I'd recommend using the terminology "Chat Completions" instead though to be more precise and clear with which API is being used.